armi/operators/operator.py

```diff
@@ -96,6 +96,7 @@ def __init__(self, cs):
         """
         self.r = None
         self.cs = cs
+        runLog.LOG.startLog(self.cs.caseTitle)
```

Comment:
Does the case that _creates_ an operator not already call `startLog()`? If it does, are there side effects of calling it twice?

Starting to feel more and more that as we drift from the world of "everything interesting that ARMI can do is part of 'running a case'," the setup for logging and verbosity and stuff needs to get closer to `context`. Sure, maybe when a case starts running, or an operator gets built, or a `Settings` object gets created or whatever, we can use `[branch]Verbosity` (and future per-`Logger` verbosity dictionary setting) to tweak the `runLog` verbosities from their defaults.

@john-science I wonder what your thoughts are on this.

Reply:
On the Framework side, I'd like to say it is a minor change to move `startLog()` from `OperatorMPI` to `Operator`.

But, honestly, I can't at this exact second. What does this change downstream?  I've been looking around, and I can't be really confident just yet.  Still looking.

Reply:
Okay, @jakehader  I have a question about your change.  In `armi.operators.factor()`, we call `getOperatorClassFromSettings`, which has this block, which is called directly from `case.py`:

```python
    if runType == RunTypes.STANDARD:                                                                                                                                                                               
        if armi.MPI_SIZE == 1:                                                                                                                                                                                     
            return Operator                                                                                                                                                                                        
        else:                                                                                                                                                                                                      
            return OperatorMPI 
```

Was this an intentional change?  It looks like we either will-or-won't get a `startLog()` in our `case` workflow, depending on if we are in an MPI run or not.

Reply:
Yeah seems like we should have a runLog if it's serial too so thats why I moved it

Reply:
Also I haven't seen a problem with calling startLog multiple times, but maybe this isn't intended?